test/sample_test.js

```diff
@@ -1,8 +1,18 @@
-var assert = require('assert');
-describe('Array', function () {
-  describe('#indexOf()', function () {
-    it('should return -1 when the value is not present', function () {
-      assert.equal([1, 2, 3].indexOf(4), -1);
-    });
-  });
-});
+var assert = require('assert')
+
+var index = require('../src/index')
+
+describe('#checkIfWord()', function() {
```

Comment:
Shouldn't this test be on its own file? (Or rename the test to match what it's testing.)